docs/metric_deletion_and_retention.md

```diff
@@ -122,6 +122,18 @@ SELECT compress_chunks(show_chunks('prom_data.container_cpu_load_average_10s', o
 
 Promscale offers full control over data retentions i.e., you can set a default data retention period as well as overwrite the default on a per-metric basis.
 
+SQL command to get the default retention policy
```

Comment:
```suggestion
Get the default retention period
```
I think we should keep this as period, since its a duration, than saying a policy.